exercises/nth-prime/uNthPrimeTest.pas

```diff
@@ -0,0 +1,49 @@
+unit uNthPrimeTest;
+
+interface
+
+uses
+  DUnitX.TestFramework;
+
+const
+  CanonicalVersion = '2.1.0';
+
+type
+  [TestFixture]
+  TNthPrimeTest = class(TObject) 
+  public
+    [Test]
+//    [Ignore('Comment the "[Ignore]" statement to run the test')]
+    procedure there_is_no_zeroth_prime;
+
+//  'Uncomment the wanted test cases attributes to run the test')]
+//    [TestCase('first prime','1,2')]
+//    [TestCase('second prime','2,3')]
+//    [TestCase('sixth prime','6,13')]
+//    [TestCase('second prime','10001,104743')]
+    procedure prime_test(const AValue1 : Integer;const AValue2 : Integer);
```

Comment:
Just occured to me after reviewing this in Delphi...  `AValue1`, should be called `aNumber` or `aInput` and `AValue2` should be called `aExpected`.  This provides more meaning to the two params of `prime_test` and provides some tie in to the canonical data.